lib/foreman_rh_cloud/engine.rb

```diff
@@ -157,37 +157,43 @@ def self.register_scheduled_task(task_class, cronline)
       end
     end
 
-    initializer 'foreman_rh_cloud.register_rex_features', :before => :finisher_hook do |_app|
+    initializer 'foreman_rh_cloud.register_rex_features', :before => :finisher_hook, :after => 'foreman_remote_execution.register_plugin' do |app|
+      app.reloader.to_prepare do
```

Comment:
The alternative seems to be to do this in a `config.to_prepare` block, and not in an initializer.

I have no idea what the differences are…